composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "php": "^7.1"
+        "php": ">=7.1"
```

Comment:
It's better to define requirement as "^7.1 || ^8.0". It would be much more strict

Reply:
https://twitter.com/nicolasgrekas/status/1263023258938548225

Reply:
Nikolas is very authoritative and smart guy, but this opinion is very controversal. I still believe that code that was not tested on some major version of the language should not support it implicitly